explorer/main.cpp

```diff
@@ -80,6 +80,20 @@ auto ExplorerMain(int argc, char** argv, void* static_for_main_addr,
           clEnumValN(ProgramPhase::All, "all",
                      "Include trace output for all phases.")));
 
+  cl::list<FileContext> allowed_file_contexts(
+      cl::desc("Select file contexts for which you want to include the trace "
+               "output"),
+      cl::values(
+          clEnumValN(
+              FileContext::Main, "trace_file_main",
+              "Include trace output for file containing the main function"),
+          clEnumValN(FileContext::Prelude, "trace_file_prelude",
+                     "Include trace output for prelude"),
+          clEnumValN(FileContext::Import, "trace_file_import",
+                     "Include trace output for imports"),
+          clEnumValN(FileContext::All, "trace_file_all",
+                     "Include trace output for all files")));
+
```

Comment:
Can you switch to supporting `--trace_file=main` etc here?

Reply:
Sure, pushing changes in the next commit.